specs/Media.md

```diff
@@ -0,0 +1,193 @@
+
+# Description
+Enable end developer to `Mute`, `Unmute` a webview content. And ability to check for the `IsMuted` and `IsCurrentlyAudible`.
+
+# Examples
+
+The following code snippet demonstrates how the Media related API can be used:
+
+## Win32 C++
+
+```cpp
+AudioComponent::AudioComponent(AppWindow* appWindow)
+    : m_appWindow(appWindow), m_webView(appWindow->GetWebView())
+{
+    //! [IsCurrentlyAudibleChanged]
+    // Register a handler for the IsCurrentlyAudibleChanged event.
+    // This handler just announces the audible state on the window's title bar.
+    CHECK_FAILURE(m_webView->add_IsCurrentlyAudibleChanged(
+        Callback<ICoreWebView2StagingIsCurrentlyAudibleChangedEventHandler>(
+            [this](ICoreWebView2Staging2* sender, IUnknown* args) -> HRESULT {
+                BOOL isAudible;
+                CHECK_FAILURE(sender->get_IsCurrentlyAudible(&isAudible));
+
+                BOOL isMuted;
+                CHECK_FAILURE(sender->get_IsMuted(&isMuted));
+
+                wil::unique_cotaskmem_string title;
+                m_webView->get_DocumentTitle(&title);
+                std::wstring result = (isAudible ? L"🔊 " : L"") + std::wstring(title.get()) +
+                                      (isMuted ? L" 🔇" : L"");
+
+                SetWindowText(m_appWindow->GetMainWindow(), result.c_str());
+                return S_OK;
+            })
+            .Get(),
+        &m_isCurrentlyAudibleChangedToken));
+    //! [IsCurrentlyAudibleChanged]
+
+    //! [IsMutedChanged]
+    // Register a handler for the IsMutedChanged event.
+    // This handler just announces the mute state on the window's title bar.
+    CHECK_FAILURE(m_webView->add_IsMutedChanged(
+        Callback<ICoreWebView2StagingIsMutedChangedEventHandler>(
+            [this](ICoreWebView2Staging2* sender, IUnknown* args) -> HRESULT {
+                BOOL isAudible;
+                CHECK_FAILURE(sender->get_IsCurrentlyAudible(&isAudible));
+                
+                BOOL isMuted;
+                CHECK_FAILURE(sender->get_IsMuted(&isMuted));
+
+                wil::unique_cotaskmem_string title;
+                m_webView->get_DocumentTitle(&title);
+                std::wstring result = (isAudible ? L"🔊 " : L"") + std::wstring(title.get()) + 
+                    (isMuted ? L" 🔇" : L"");
+
+                SetWindowText(m_appWindow->GetMainWindow(), result.c_str());
+                return S_OK;
+            })
+            .Get(),
+        &m_isMutedChangedToken));
+    //! [IsMutedChanged]
+}
+
+//! [Mute]
+// Mute the current window and show a mute icon on the title bar
+ void AudioComponent::Mute()
+ {
+     m_webView->Mute();
+ }
+//! [Mute]
+
+//! [Unmute]
+// Unmute the current window and hide the mute icon on the title bar
+ void AudioComponent::Unmute()
+ {
+     m_webView->Unmute();
+ }
+ //! [Unmute]
+```
+
+## .NET and WinRT
+
+```c#
+    void MuteCmdExecuted(object target, ExecutedRoutedEventArgs e)
```

Comment:
The .NET sample code should be doing the same thing as the C++ sample code

Reply:
updated